src/System.IO.Pipes/src/System/IO/Pipes/NamedPipeServerStream.Unix.cs

```diff
@@ -99,9 +99,7 @@ public void WaitForConnection()
                 TranslateFlags(_direction, _options, _inheritability), 
                 (int)Interop.libc.Permissions.S_IRWXU);
 
-            // Ignore _inBufferSize and _outBufferSize.  They're optional, and the fcntl F_SETPIPE_SZ for changing 
-            // a pipe's buffer size is Linux specific.
-
+            InitializeBufferSize(serverHandle, _outBufferSize); // there's only one capacity on Linux; just use the out buffer size
```

Comment:
Interestingly, that's actually what I wrote initially :smile:  But then I hesitated because I wasn't actually sure the max would be desirable... maybe, for example, the user was actually trying to lower the size?  Since the out size is already what's being passed to the base (on Windows as well), I decided to just go with that.  Let me know if you still think I should change it to a max... I don't feel strongly about it either way, so if you think it's the right thing to do, just let me know.